platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -7,116 +7,133 @@
 version: 1
 
 ##############################
-## Application Configuration
+# Application Configuration
 ##############################
 
+#
+# The default language of the application
+# The supported languages defined by RFC4646 (https://datatracker.ietf.org/doc/html/rfc4646)
+#
+languages: en
```

Comment:
What does this config do?

Reply:
The port `9083` is currently used for Spring Boot actuator. The anchor should refer to the Spring documentation.

Reply:
The `language: en` is used to specify the languages supported in SEP server. It is legacy. Maybe we should review it in the next major release.